app-frontend/src/assets/styles/sass/components/_filter.scss

```diff
@@ -16,6 +16,9 @@
     font-size: 1.2rem;
     flex-shrink: 0;
   }
+  .selectize-dropdown {
+      width: calc(70%);
```

Comment:
What's the purpose of wrapping this in `calc`? Without calc it seems to be displayed the same.